impeller/renderer/vertex_descriptor.cc

```diff
@@ -36,18 +40,26 @@ size_t VertexDescriptor::GetHash() const {
   for (const auto& input : inputs_) {
     fml::HashCombineSeed(seed, input.GetHash());
   }
+  for (const auto& layout : layouts_) {
+    fml::HashCombineSeed(seed, layout.GetHash());
+  }
   return seed;
 }
 
 // |Comparable<VertexDescriptor>|
 bool VertexDescriptor::IsEqual(const VertexDescriptor& other) const {
-  return inputs_ == other.inputs_;
+  return inputs_ == other.inputs_ && layouts_ == other.layouts_;
```

Comment:
This doesn't actually seem like the right comparison but unsure if its actually used.